package.json

```diff
@@ -9,5 +9,13 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.2",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@angular/animations": "^10.0.2",
+    "@angular/common": "^10.0.2",
+    "@angular/core": "^10.0.2",
+    "@angular/platform-browser": "^10.0.2",
+    "@angular/router": "^10.0.2",
+    "ngx-owl-carousel-o": "^3.0.1"
```

Comment:
We don't need to modify the `package.json` in the root directory -- this is just for linting.

Reply:
Done!